bin/ruby-build

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env bash
 #
-# Usage: ruby-build [-kvp] <definition> <prefix>
+# Usage: ruby-build [-kpv46] <definition> <prefix>
```

Comment:
I wouldn't add further documentation for `-46` because those are flags that I've never approved and am not sure what are they even used for. https://github.com/rbenv/ruby-build/pull/599#issuecomment-151804316